Repository/Context/ApplicationDbContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Application.Interfaces;
+using Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Repository
+{
+    public class ApplicationDbContext : DbContext, IApplicationDbContext
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
+        {
+
+        }
+
+        public DbSet<Booking> Bookings { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
why exceptions? before creating first migration?

Reply:
Would it not be better to create first migrations after at least half of the entities have been added? Entities are not ready yet. They are assigned to others but not merged yet.

Reply:
If you leave it as is you will have to change it later, so change it now to not make an extra work for future